tests/unit/amo/pages/TestUserProfileEdit.js

```diff
@@ -1659,4 +1660,19 @@ describe(__filename, () => {
 
     sinon.assert.calledWith(dispatchSpy, errorHandler.createClearingAction());
   });
+
+  it('renders a FxA management link', () => {
```

Comment:
would it be a little bit better to say when this link is present? 
something like: `renders a FxA management link when it's the user's own profile`